hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,20 @@
+RELEASE_TYPE: minor
+
+This release improves support for the SupportsOp protocols from the :mod:`python:typing` 
+module when using on :func:`~hypothesis.strategies.from_type` as outlined in :issue:`2292`.
+The following types now generate more wide ranging and correct strategies when called 
```

Comment:
I'd say "generate much more varied strategies", since the old one was correct too :slightly_smiling_face:

Reply:
`SupportsRound` was  the only one that was actually incorrect i think. I will change it.